plugwise_usb/constants.py

```
@@ -7,349 +7,84 @@
-SED_AWAKE_STARTUP: Final = (
-    2  # SED awake after restart, e.g. after reinserting a battery
-)
-SED_AWAKE_STATE: Final = 3  # SED awake to report state (Motion / Temperature / Humidity
-SED_AWAKE_UNKNOWN: Final = 4  # TODO: Unknown
-SED_AWAKE_BUTTON: Final = 5  # SED awake due to button press
 
 # Max timeout in seconds
-MESSAGE_TIME_OUT: Final = 15  # Stick responds with timeout messages after 10 sec.
-MESSAGE_RETRY: Final = 2
+# Stick responds with timeout messages within 10s.
+STICK_TIME_OUT: Final = 11 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)